examples/tutorials/analysis-3d/event_sampling.py

```diff
@@ -309,8 +309,9 @@
 #
 
 primary_hdu = fits.PrimaryHDU()
-hdu_evt = fits.BinTableHDU(events.table)
-hdu_gti = fits.BinTableHDU(dataset.gti.table, name="GTI")
+hdu_gti = dataset.gti.to_table_hdu()
+hdu_evt = events.to_table_hdu()
+hdu_evt.header.update(observation.pointing.to_fits_header())
```

Comment:
Shouldn't this be moved into the event sampler code itself?
It is currently failing in make docs.

Reply:
This should be added to EventList.to_table_hdu directly, right?